pkgs/misc/lilypond/default.nix

```diff
@@ -62,4 +63,8 @@ stdenv.mkDerivation rec {
     maintainers = with maintainers; [ marcweber yurrriq ];
     platforms = platforms.all;
   };
+
+  FONTCONFIG_FILE = lib.optional stdenv.isDarwin (makeFontsConf {
```

Comment:
meta should be last according to the contributing guide

Reply:
I can't seem to find any reference to that, but I'm known to miss the obvious sometimes. Could you share a link?